Project-Aurora/Project-Aurora/Profiles/Stardew Valley/GSI/GameState_StardewValley.cs

```diff
@@ -0,0 +1,29 @@
+using Aurora.Profiles.Generic.GSI.Nodes;
+using Aurora.Profiles.StardewValley.GSI.Nodes;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Aurora.Profiles.StardewValley.GSI {
+    public class GameState_StardewValley : GameState {
+        public ProviderNode Provider => NodeFor<ProviderNode>("Provider");
+
+        public WorldNode World => NodeFor<WorldNode>("World");
+        public PlayerNode Player => NodeFor<PlayerNode>("Player");
+        public InventoryNode Inv => NodeFor<InventoryNode>("Inv");
```

Comment:
you can just call this Inventory, Inv sounds a bit weird

Reply:
Changed